flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/harness/RankHarnessTest.scala

```diff
@@ -43,6 +43,7 @@ import java.util.concurrent.ConcurrentLinkedQueue
 import scala.collection.mutable
 
 @RunWith(classOf[Parameterized])
+@Ignore
```

Comment:
Why this?

Reply:
Sorry, reverted it. It fails regularly and have temporarily disabled it locally but forgot to remove.
Many thx for checking thoroughly and catching it!!!